app/uk/gov/hmrc/hmrcemailrenderer/templates/twowaymessage/twoWayMessagingTemplates.scala

```diff
@@ -23,7 +23,7 @@ import uk.gov.hmrc.hmrcemailrenderer.templates.ServiceIdentifier.TwoWayMessaging
 object twoWayMessagingTemplates {
   val templates = Seq(
     MessageTemplate.create(
-      templateId = "twoWayMessageReceived",
+      templateId = "2wsm-customer",
       fromAddress = FromAddress.noReply("twoWayMessingRecieved"),
```

Comment:
@matthewgroom what should this be?

Reply:
It's perfect as it is. As long as the 'templateId' is the same name as the one inside the 'ServiceIdentifier' file, then it's good to go